openshift/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:centos7
+
+MAINTAINER Martin Gencur <mgencur@redhat.com>
+
+#ENV RADARGUN_VERSION=
+#ENV DESCRIPTION=
+
+RUN yum -y update && \
+        yum -y install unzip java-1.8.0-openjdk-devel maven openssh-server which hostname ls ps grep pwd id rsync && \
+        yum clean all -y
+
+RUN mkdir /opt/radargun /opt/radargun-data 
+COPY RadarGun-3.0.0-SNAPSHOT /opt/radargun/
```

Comment:
where `RadarGun-3.0.0-SNAPSHOT` comes from? It's not present there yet

Reply:
It is copied from ${RADARGUN_HOME}/target/distribution/RadarGun-3.0.0-SNAPSHOT into the current directory before building the image. This is in openshift.sh in function build_radargun_image.